department-of-veterans-affairs/caseflow
# Create Redux Action for Adding New Appeal-Related Tasks

[](url)
Resolves [Create Redux Action to Add Task to Corresponding Appeal Info (In Store)](https://jira.devops.va.gov/browse/APPEALS-34935)

# Description
Adds a new Redux action that will create a new appeal-related task in the Redux store and add it to the array of new tasks to be created at the end of the intake.

## Acceptance Criteria

## Testing Plan
1. Verify the appeal intake form loads properly, esp. step 2.
2. Verify initial state of `newAppealRelatedTasks` (empty array) Redux store.

# Screenshots
Here is a screenshot of the Redux store being updated via a custom action dispatch in the Redux dev tools:
<img width="2554" alt="Screenshot 2023-11-21 at 12 05 48" src="https://github.com/department-of-veterans-affairs/caseflow/assets/2421172/b7768c97-8181-47a9-9fb3-010d9790778e">
